### PR TITLE
[TARGET] each option of target str should only contain one '='

### PR DESCRIPTION
**Pull request by windclarion**
src/target/target_id.cc ParseAttrsFromRawString L222:
if ((pos = FindUniqueSubstr(s, "=")) != -1)
require option contains only one '='

The CHECK code in FindUniqueSubstr will cause build fail.

Signed-off-by: windclarion <windclarion@gmail.com>

### CODE REVIEW

File: python/tvm/target/target.py [hunk 1]
```diff
@@ -194,21 +194,22 @@ def arm_cpu(model='unknown', options=None):
...
+        "rasp4b":    ["-model=bcm2711", "-target=arm-linux-gnueabihf", "-mattr=+neon"],
+        "rk3399":    ["-model=rk3399", "-target=aarch64-linux-gnu", "-mattr=+neon"],
+        "pynq":      ["-model=pynq", "-target=armv7a-linux-eabi", "-mattr=+neon"],
+        "ultra96":   ["-model=ultra96", "-target=aarch64-linux-gnu", "-mattr=+neon"],
     }
     pre_defined_opt = trans_table.get(model, ["-model=%s" % model])
 
     opts = ["-device=arm_cpu"] + pre_defined_opt
     opts = _merge_opts(opts, options)
+    print('opts=', opts)
```
Comment (siju-samuel) [inline python/tvm/target/target.py:212]: Remove the print statement

Comment (windclarion) [reply python/tvm/target/target.py]: sorry! done!

**Comment by junrushao1994** [comment]
Ah I see, I am a bit surprised the typo is not found before. BTW, please rebase to master, and change “-target” to “-mtriple”. Thanks!

**Comment by windclarion** [comment]
done!  old code works ok before "[Target] Migrate data structure of TargetNode (#5960)", the commit add a requirement in src/target/target_id.cc. FindUniqueSubstr require options string has only one '=', we didn't force the limit before that commit.

**Comment by junrushao1994** [comment]
@windclarion Looks like a flaky test. Could you file an empty commit using `git commit --allow-empty -m "retrigger"` to retrigger the CI?

**Comment by windclarion** [comment]
@ZihengJiang I try to add test, but if I feed the option with a string which contains two '=', target_id.cc  FindUniqueSubstr  CHECK will abort,  so  I don't know how can I test the abort operation, it's not a exception which I can catch, it's just abort. Can you give me   some advice or tell me similar test file which can test the CHECK?

**Comment by junrushao1994** [comment]
@windclarion use pytest.raises: https://stackoverflow.com/questions/23337471/how-to-properly-assert-that-an-exception-gets-raised-in-pytest

**Comment by windclarion** [comment]
@junrushao1994  What I have said  maybe not very clear,  I mean  I can't catch a exception in python code, because C++ CHECK just abort, the abort function didn't throw a exception.

**Comment by windclarion** [comment]
@junrushao1994  I  add a  small test which just test target wether can be created, and I just check the target shouldn't be None.

**Comment by windclarion** [comment]
today's test is flaky, I always failed at unexpected position

**Comment by junrushao1994** [comment]
Yeah seems like another flaky test...Could you re-trigger? Thanks!

**Comment by junrushao1994** [comment]
I see. There is a flaky test in the recent week: https://github.com/apache/incubator-tvm/issues/6009. Don't worry, let's wait until it is fixed.

**Comment by ZihengJiang** [comment]
@windclarion Could you try to build a small program like`A+B` with every target for testing the code generation? ( you can jump some target like cuda if it requires extra efforts)

**Comment by junrushao1994** [comment]
The flaky test is resolved in #6014 :-)

**Comment by windclarion** [comment]
@ZihengJiang many other tests has tested tvm.build, ex. tests\python\integration\test_reduce.py test_reduce_prims Line64 has checked metal, vulkan, cuda, opencl, rocm
